spec/directedGraph_test.js

```diff
@@ -0,0 +1,50 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+
+import DirectedGraph from '../src/DirectedGraph'
+
+chai.use(chaiChange)
+
+describe('DirectedGraph', () => {
```

Comment:
same as above for response : ). Needed to see how I wanted it to look before i started coding.